resources/assets/js/components/JobCard.tsx

```diff
@@ -29,7 +29,7 @@ const StatusPill: React.FC<StatusPillProps> = ({ text, status }) => (
 export interface JobCardProps {
   id: number;
   activity: Activity;
-  applicants: number;
+  applicants: number | null;
```

Comment:
We should probably rename this to something more explicit like `totalApplications` or `numOfApplications`.

Reply:
Nvm, new changes made on #4656 removing the need for this.